src/curl_async.c

```diff
@@ -224,6 +226,20 @@ void swupd_curl_parallel_download_set_callbacks(void *handle, swupd_curl_success
 	h->free_cb = free_cb;
 }
 
+void swupd_curl_parallel_download_set_progress_callbacks(void *handle, swupd_curl_progress_cb progress_cb, struct download_progress *progress)
```

Comment:
s/struct download_progress *progress/void *data/

Reply:
I will make the change